src/panel/category/CategoryPanel.jsx

```diff
@@ -98,7 +101,7 @@ export default class CategoryPanel extends React.Component {
     }
   }
 
-  fetchData = async () => {
+  fetchData = debounce(async () => {
```

Comment:
`debounce` always adds a delay after the last event before actually calling the function, so the app appears less responsive in all cases. For example there is a weird wait when we click a category button and the data appears. 
[`throttle`](https://lodash.com/docs/4.17.15#throttle) is more adapted to the task.

Reply:
After discussion, we agreed to still use debounce. I edited the debounce delay to 100ms in order to improve the UX